types/database.ts

```diff
@@ -29,19 +29,31 @@ export interface VolunteerLogData {
   createdAt: Date;
 }
 
-export interface VolunteerApplicationData {
+export interface VolunteerProgramApplicationData {
   _id: mongoose.Types.ObjectId;
   userId: mongoose.Types.ObjectId;
-  isApproved?: boolean;
-  emergencyContact: {
-    firstName: string;
-    lastName: string;
-    phone: string;
-    relationship: string;
-  };
-  workStatus?: string;
-  employer?: string;
-  opportunities?: Array<string>;
+  programId: mongoose.Types.ObjectId;
+  formData: mongoose.Schema.Types.Mixed;
```

Comment:
same as above. we need formData to adhere to an automatically generated type (RIF will have its own type, RFR its own type, etc.)